.github/workflows/spm.yml

```diff
@@ -57,16 +80,24 @@ jobs:
     # Don't run on private repo unless it is a PR.
     if: (github.repository == 'Firebase/firebase-ios-sdk' && github.event_name == 'schedule') || github.event_name == 'pull_request'
 
-    runs-on: macos-12
     strategy:
       matrix:
+        # Full set of Firebase-Package tests only run on iOS. Run subset on other platforms.
         target: [tvOS, macOS, catalyst]
-        # Full set of Firebase-Package tests only run on iOS.
+        os: [macos-12, macos-13]
+        include:
+          - os: macos-12
+            xcode: Xcode_14.2
+          - os: macos-13
+            xcode: Xcode_15.0.1
```

Comment:
(No need to address this in this PR) Something we could consider down the line is environment variables for our latest Xcode version (`Xcode_15.0.1`) and previous (`Xcode_14.2`) if they could be set for the whole repo.

Reply:
Agreed, this is just one of several things in the Actions jobs that would be good to parameterize/templatize